### DISCUSSION TITLE
你们未来会支持 Cloudflare Workers 服务端部署吗？ || Will you support Cloudflare Workers server-side deployment in the future?

### DISCUSSION THREAD

**Discussion by AkarinLiu** [walinejs/waline#2584] [General]
我想把 Waline 部署在 Cloudflare 上面，因为 Cloudflare Workers 有 D1 无服务器数据库，而且每天有 1000 次写入、删除和列出操作，你们未来会支持它吗？

---
I want to deploy Waline on Cloudflare as Cloudflare Workers has D1 serverless database and has 1000 write, delete and list operations per day, will you support it in the future?

**Comment by AkarinLiu** [comment]
![无标题](https://github.com/walinejs/waline/assets/58081491/5cc42e2a-1939-4978-a081-16b7b9ea6fbb)


---
![Untitled](https://github.com/walinejs/waline/assets/58081491/5cc42e2a-1939-4978-a081-16b7b9ea6fbb)

**Comment by lizheming** [comment]
CF Workers 的架构和 Node.js 不是特别适配，应该不会直接支持。或许可以重新基于同样的 API 写一个 CF Workers 的 server 版本出来。

---
The architecture of CF Workers is not particularly suitable for Node.js and should not be directly supported. Maybe we can re-write a server version of CF Workers based on the same API.

**Reply by xiao8hello** [reply]
即使使用兼容性标志 `nodejs_compat` 也无法运行吗？

---
Does it not work even with the compatibility flag `nodejs_compat`?

**Reply by lizheming** [reply]
去年这个问题时cf worker还没有推出node的适配。今年需要再看一下，不确定是否可以

---
Last year, cf worker has not yet launched node adaptation. I need to take a look this year, not sure if

**Reply by xiao8hello** [reply]
现在好像已经出到 `nodejs_compat_v2` 了

---
Now it seems that `nodejs_compat_v2` has been released

**Comment by wuyilingwei** [comment]
https://github.com/wuyilingwei/Waline_On_Worker
尝试搓了一个后端实现+前端直接借用Waline的项目

---
https://github.com/wuyilingwei/Waline_On_Worker
I tried to create a back-end implementation + the front-end directly borrowed Waline's project
